README.md

```diff
@@ -1,12 +1,12 @@
 [![Read The Docs Status](https://readthedocs.org/projects/ufs-weather-model/badge/?badge=latest)](http://ufs-weather-model.readthedocs.io/)
 
-# ufs-weather-model
+# ufs-model
```

Comment:
please do not use ufs-model. the unified repo name has been decided yet.

Reply:
Done